src/pyextremes/eva.py

```diff
@@ -217,9 +217,15 @@ def AIC(self) -> float:
 
     def test_ks(self, significance_level: float = 0.05) -> KolmogorovSmirnov:
         return KolmogorovSmirnov(
-            extremes=self.extremes,
+            extremes=self.extremes_transformer.transformed_extremes,
             distribution=self.distribution.distribution,
-            fit_parameters=self.model.fit_parameters,
+            fit_parameters={
+                **self.model.fit_parameters,
+                **{
+                    'loc' if k=='floc' else
+                    k: v for k, v in model.distribution.fixed_parameters.items()
+                }
```

Comment:
I think you meant `self.model`.

As for stripping `f` from fixed parameters like `floc` you can do something like this (we know that all fixed parameters in `scipy` must have prefix `f`):

https://github.com/georgebv/pyextremes/blob/a0b4baab5f1263691e6e32f3e26c14877d9a2f39/src/pyextremes/models/distribution.py#L99-L101

or you can just use the property directly like so:

https://github.com/georgebv/pyextremes/blob/a0b4baab5f1263691e6e32f3e26c14877d9a2f39/src/pyextremes/models/distribution.py#L218-L223

Reply:
Thank you -- the PR is updated to use `self._fixed_parameters` directly